assembler/manual.html

```diff
@@ -548,10 +550,10 @@ <h4>Pipeline options</h4>
     <code>--restart-from &lt;check_point></code><br>
     &nbsp;&nbsp;&nbsp;&nbsp;Restart SPAdes run from the specified output folder starting from the specified check-point. Check-points are:
     <ul>
-        <li><code>ec</code> &ndash; start from error correction</li>
-        <li><code>as</code> &ndash; restart assembly module from the first iteration</li>
+        <li><code>ec</code> &ndash; start from error correction (BayesHammer or IonHammer)</li>
+        <li><code>as</code> &ndash; restart assembly module from the first iteration (from the lowest k-mer size)</li>
         <li><code>k&lt;int></code> &ndash; restart from the iteration with specified k values, e.g. k55</li>
-        <li><code>mc</code> &ndash; restart mismatch correction</li>
+        <li><code>mc</code> &ndash; restart mismatch correction of assembled contigs (MismatchCorrector which uses BWA)</li>
```

Comment:
See above

Reply:
I am maybe sloppy but it took me a while what you mean under "mismatch correction" here and there. This change makes it clear what it does and when it happens.